src/components/Templates/Home/Home.component.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { IProduct } from '../../../types/models/Product';
+import { ProductCardContainer } from '../../Molucules/ProductCardConatiner';
+import { Box } from '@mui/material';
+import { Categories } from '../../Molucules/Categories';
+
+const Home = () => {
+  const [products, setProducts] = useState<IProduct[]>([]);
+  useEffect(() => {
+    axios
+      .get('https://fakestoreapi.com/products')
+      .then((res) => {
+        const products = res.data.filter(
+          (prod: IProduct) =>
+            prod.category === "men's clothing" ||
+            prod.category === "women's clothing"
+        );
+        setProducts(products);
+      })
+      .catch((err) => {
+        console.log('error', err);
+      });
+  }, []);
+
+  return (
+    <Box width={1} maxWidth={1100} mx='auto' mt={4}>
+      {products.length > 0 && <ProductCardContainer products={products} />}
+      <Categories />
+    </Box>
+  );
+};
```

Comment:
@siroshan Let's create a `pages` folder inside `templates` and move this inside it so that we can identify page templates quickly!

Also I don't see "Flash Sale" heading being passed here—we can use the `SectionLayout` here to wrap this `ProductCardContainer` and add the `heading` of the section